subprojects/docs/src/snippets/testing/junit-test-suite-categories/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+plugins {
+    java
+    `jvm-test-suite`
+}
+
+repositories {
+    mavenCentral()
+}
+
+testing {
+    suites {
+        named<JvmTestSuite>("unitTest") { // FIXME TestSuite with name 'unitTest' not found.
+            useJUnit()
+            dependencies {
+                implementation("junit:junit:4.13")
+            }
+            targets {
+                all {
+                    // tag::test-categories[]
+                    testTask.configure {
+                        options { // FIXME Ugly unsafe casts. Do we need to qualify the type of options below?
```

Comment:
ugh, I didn't think about how Kotlin would hate this.

We can just use `useJUnit {` here instead. That should get rid of the cast